setup.cfg

```diff
@@ -0,0 +1,4 @@
+[pep8]
```

Comment:
MAGStock is PEP-8 compliant, with a few exceptions set aside.  Most of the exceptions listed are the standard recommended exceptions, plus a few that I added.  For example, it's usually a PEP-8 violation to say ``Attendee.foo == None`` instead of ``Attendee.foo is None`` but SQLAlchemy actually requires us to use ``==`` for queries, so we just turned that check off, along with a few others like it.